feeds/src/components/vis/graph/GraphTooltip.tsx

```diff
@@ -1,14 +1,30 @@
 import React, { useEffect, useState } from 'react'
 import { connect } from 'react-redux'
 import { humanizeUnixTimestamp } from 'utils'
+import { AppState } from 'state'
+import { FeedConfig } from 'feeds'
 
-const positionTypes = {
+export interface Props {
+  config: FeedConfig
+  tooltip: any
+}
+
+interface PositionTypesItem {
+  x: number
+  y: number
+}
+
+interface PositionTypes {
+  [key: string]: PositionTypesItem
+}
+
+const positionTypes: PositionTypes = {
   oracle: { x: 10, y: 10 },
   contract: { x: 20, y: 20 },
 }
 
-const Tooltip = ({ options, tooltip }) => {
-  const [position, setPosition] = useState({})
+const Tooltip: React.FC<Props> = ({ config, tooltip }) => {
+  const [position, setPosition] = useState<any>({})
```

Comment:
```suggestion
  const [position, setPosition] = useState<PositionTypes>({})
```

Reply:
`position` hook is used in inline styles (left, right) so it is required to have a new interface.